libs/cpp-ledger/ledger/utils.cpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "cpp-ledger/ledger/utils.hpp"
+
+#include <wchar.h>
+#include <codecvt>
+#include <cstring>
+#include <locale>
+
+namespace ledger {
+
+  void copyStr(char *dest, const char *src) {
+    if (src == nullptr) {
+      return;
+    }
+
+    if (dest != nullptr) {
+      delete[] dest;
+    }
+
+    dest = new char[std::strlen(src) + 1];
+    std::strcpy(dest, src);
+  }
+
+  void copyStr(wchar_t *dest, const wchar_t *src) {
+    if (dest != nullptr) {
+      delete[] dest;
+    }
+
+    dest = new wchar_t[std::wcslen(src) + 1];
```

Comment:
can `src` be null?  
same memory leak.

Reply:
Yes, missed `scr` checking.

There is no memory leak.